kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -301,6 +301,43 @@ struct GetCategories {
   }
 };
 
+struct GetLatestCategoryInfo {
```

Comment:
I was thinking a clearer name may be `GetLatestBlockIDContainingCategory`. What do you think?

Reply:
Ah, we also return the updates themselves. So maybe then `GetLatestCategoryUpdates`?

Reply:
done